buildenv/jenkins/JenkinsfileBase

```diff
@@ -696,11 +696,15 @@ def post(output_name) {
             def testenvFile = "aqa-tests/testenv/testenv.properties"
             def shaFile = "aqa-tests/TKG/SHA.txt"
             def AQACertFile = "aqa-tests/TKG/AQACert.log"
+		  def TAPzipFile = "aqa-tests/TKG/TAPFiles.zip"
 
             //call the archive function for each file
             archiveFile(testenvFile)
-	    archiveFile(shaFile)
+            archiveFile(shaFile)
             archiveFile(AQACertFile)
+		  // TODO: how to push the tap files to TAPzipFile with the same archiveFile function?
+		  // archiveArtifacts artifacts: "**/*.tap", fingerprint: true, allowEmptyArchive: true
+		  archiveFile(TAPzipFile)
```

Comment:
Please remove line 695 - 699 and update line 702 - 707 to directly pass in the file as the parameter.